models/global_upgrade.py

```diff
@@ -1,4 +1,15 @@
 from models.common import WarframeObj
+from collections import defaultdict
 
 class GlobalUpgrade(WarframeObj):
-    pass
\ No newline at end of file
+    def __init__(self, json_global_upgrade : dict):
+        json_global_upgrade = defaultdict(lambda : None, json_global_upgrade)
+        self.start = json_global_upgrade['start']
```

Comment:
Is this a time thing potentially? As well as "end" below. Check the output and see if it is....